### DISCUSSION TITLE
Reindex without extraCharsToIndex

### DISCUSSION THREAD

**Discussion by DeveloperNamedMax** [sepinf-inc/IPED#2070] [Q&A] [answered]
At the moment i have indexed 4TB and since i have some documents that have inside document numbers in format "127-8/12.14" in order to make that search possible i added `extraCharsToIndex = -/.` to conf/IndexTaskConfig.txt.

Therefore my index at the moment contains special characters. Although it made a job easier when documents need to be searched using document numbers it has made normal search more difficult. 

Is there a way to remove special characters from index without indexing everything again? 

Thanks in advance!

**Comment by lfcnassif** [comment, marked as answer]
Unfortunately that's not possible. I have already considered to create some option in UI to change the default StandardQueryParser to:

https://lucene.apache.org/core/7_1_0/queryparser/org/apache/lucene/queryparser/complexPhrase/ComplexPhraseQueryParser.html

Would it help your use case?

**Reply by DeveloperNamedMax** [reply]
The ComplexPhraseQuery would help with all the use cases i currently have.

**Reply by lfcnassif** [reply]
Just created https://github.com/sepinf-inc/IPED/issues/2072
